syz-cluster/controller/api.go

```diff
@@ -35,11 +35,13 @@ func (c ControllerAPI) Mux() *http.ServeMux {
 	mux := http.NewServeMux()
 	mux.HandleFunc("/sessions/{session_id}/series", c.getSessionSeries)
 	mux.HandleFunc("/sessions/{session_id}/skip", c.skipSession)
+	mux.HandleFunc("/sessions/upload", c.uploadSession)
 	mux.HandleFunc("/series/{series_id}", c.getSeries)
 	mux.HandleFunc("/builds/last", c.getLastBuild)
 	mux.HandleFunc("/builds/upload", c.uploadBuild)
 	mux.HandleFunc("/tests/upload", c.uploadTest)
 	mux.HandleFunc("/findings/upload", c.uploadFinding)
+	mux.HandleFunc("/series/upload", c.uploadSeries)
```

Comment:
Wdyt about https://github.com/google/keep-sorted here?

Reply:
I've sorted the handlers list, but I don't feel like integrating it with the linter as part of this changeset.